Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -172,6 +172,22 @@
 		</Target>
 	</ECRelationshipClass>
 
+	<ECEntityClass typeName="Catchment" modifier="Sealed" displayLabel="Rainfall Set">
+		<BaseClass>anlyt:AnalyticalElement</BaseClass>
+		<ECProperty propertyName="Area" typeName="double" displayLabel="Area" category="HydraulicData" kindOfQuantity="rru:AREA"/>
+		<ECProperty propertyName="Tc" typeName="double" displayLabel="Time of Concentration" category="HydraulicData" kindOfQuantity="AECU:TIME"/>
+		<ECProperty propertyName="RationalC" typeName="double" displayLabel="Rational C" category="HydraulicDAta"/>
```

Comment:
The EC System supports unitless KoQ (it has a "ONE" unit: https://github.com/iTwin/bis-schemas/blob/85130ece5999e4eb6156821303948c97c0de2b26/System/Units.ecschema.xml#L601 ... or a "COEFFICIENT" unit: https://github.com/iTwin/bis-schemas/blob/85130ece5999e4eb6156821303948c97c0de2b26/System/Units.ecschema.xml#L602). Neither AecUnits nor RoadRailUnits have any KoQs based on them.

I wonder if we should add a KoQ to RoadRailUnits to be able to format all "Drainage coefficients" together? e.g.
<KindOfQuantity typeName="DRAINAGE_COEFFICIENT"     displayLabel="Drainage Unitless Coefficient"     persistenceUnit="u:COEFFICIENT"                 relativeError="0.001"  presentationUnits="f:DefaultRealU(3)[u:COEFFICIENT]"/>

Reply:
this is kind of why I thought it made sense for hydraulics to have its own set of KOQs... 
1.  I can adjust them to my specific use cases.  
2. For example 
     -  i need a land_area KOQ 
     -  a KOQ for shape  length parameters (for example elevations, vs pipe diameters and so on)
     - a rational c coefficient - might have different precision than SCS CN precision and so on...
3.  if I make a change it won't cause problems for other folks

whatever we decide - we can make adjustments to whatever units -and apply those whenever they are successfully integrated

Reply:
We'll worry about this coefficient/unitless KoQ at a later point...